test/TesterInternal/GeoClusterTests/MultiClusterRegistrationTests.cs

```diff
@@ -9,6 +9,7 @@
 using Tests.GeoClusterTests;
 using Xunit;
 using Xunit.Abstractions;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
```

Comment:
bad merge? looks like you were already using xUnit asserts but you went back to MSTest asserts (which we are slowly moving off). Tester was completely updated to use xUnit. TesterInternal is still in flux